### ISSUE TITLE
Simple AIA Fido request failing

### ISSUE THREAD

**Issue by dstansby** [sunpy/sunpy#5528]
```python
import astropy.units as u

from sunpy.net import Fido
from sunpy.net import attrs as a

aia = (a.Instrument.aia &
       a.Sample(24 * u.hour) &
       a.Time('2011-01-01', '2011-01-02'))

wave = a.Wavelength(301 * u.Angstrom, 310 * u.Angstrom)
result = Fido.search(wave & aia)
```
returns no results, and raises the warning:

```
WARNING: SunpyUserWarning: VSO-C500 :soap:Server : ERROR:  relation "vso.aia_synoptic2" does not exist
LINE 12: FROM vso.aia_synoptic2
              ^
 [sunpy.net.vso.vso]
```

This is what's causing our doc build to fail at the moment. Running a query for AIA data on the same date on the VSO website also fails, so I suspect this is a VSO issue?

**Comment by wtbarnes** [comment]
Per the chat today, this is apparently because of some hardware maintenance at the VSO last week. https://matrix.to/#/!MeRdFpEonLoCwhoHeT:matrix.org/$1632069913112438Mwcdv:matrix.org?via=matrix.org&via=openastronomy.org&via=cadair.com

**Comment by ejm4567** [comment]
Actually, the issue is there is no AIA NRT data prior to 5/11/12. Hence the error message.  JSOC gets pulled in because it matches the time and instrument parts of the query.

We've added this use case to our ticket for improving error messages in the VSO, so you'll get a more informative error message in the future.

**Comment by nabobalis** [comment]
Thanks @ejm4567.

**Comment by wtbarnes** [comment]
@ejm4567 The above `Fido` query should not be asking for NRT data. Is there some reason that it might be hitting the NRT data?

**Comment by Cadair** [comment]
here's the full debug output:
```
DEBUG: VSO Request:
 <soap-env:Envelope xmlns:soap-env="http://schemas.xmlsoap.org/soap/envelope/" xmlns:VSO="http://virtualsolar.org/VSO/VSOi">
  <soap-env:Body>
    <VSO:Query>
      <body>
        <block>
          <time>
            <start>20110101000000</start>
            <end>20110102000000</end>
          </time>
          <instrument>AIA</instrument>
          <wave>
            <wavemin>301.0</wavemin>
            <wavemax>310.0</wavemax>
            <waveunit>Angstrom</waveunit>
          </wave>
          <sample>86400.0</sample>
        </block>
      </body>
    </VSO:Query>
  </soap-env:Body>
</soap-env:Envelope>
 [sunpy.net.vso.zeep_plugins]
DEBUG: VSO Response:
 <soap:Envelope xmlns:VSO="http://virtualsolar.org/VSO/VSOi" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <soap:Body>
    <VSO:QueryResponse>
      <body>
        <provideritem>
          <provider>JSOC</provider>
          <record/>
          <version xsi:nil="true"/>
          <error>VSO-C500 :soap:Server : ERROR:  relation "vso.aia_synoptic2" does not exist
LINE 12: FROM vso.aia_synoptic2
              ^
</error>
        </provideritem>
      </body>
    </VSO:QueryResponse>
  </soap:Body>
</soap:Envelope>
```

It looks like this is an error in the depths of VSO to me.

**Comment by ejm4567** [comment]
If you pass only a date range and instrument = AIA, then both JSOC and SDAC_AIA will respond. 
JSOC serves out just SDO data, SDAC AIA serves out AIA NRT.  Hence JSOC matches the initial query, but doesn't have the synoptic data. 

If you add provider = SDAC you'll get the expected "No matching records" for dates < 2012. If you specify provider = JSOC, you'll get the expected SDO data and no NRT data.

We're working on providing better error messages in this case since the current one is rather unhelpful IMHO.

**Comment by Cadair** [comment]
This is a new failure mode, this query used to work. I don't see why the query would fail if it can't be serviced by a single provider?

**Comment by ejm4567** [comment]
Hmmm  so the query w/2011 dates worked in the past ?  Let me check the JSOC Data Provider code on the new server

**Comment by Melmot** [comment]
Yeah, I'm having a similar but somewhat more severe problem. Previously I was able to download the 2010-2011 AIA data with Fido all right. Then, about a week ago, nearly half of the downloads began to fail (a bit less so with JSOC provider). And now I can barely download a couple of files in a hundred attempts, even for the latest 2021 data, no matter which provider I choose. I typically get a "Max retries exceeded" error, or, mostly previously, "relation vso.aia_synoptic2 does not exist" same as above. A possibility is, though, I could have updated the Sunpy package just around that time.
